docs/deployment/getting_started_locally.md

```diff
@@ -94,12 +94,6 @@ Afterwards, the Gardener resources will be deployed into the cluster.
 
 ## Creating a `Shoot` Cluster
 
-You can wait for the `Seed` to be ready by running:
-
-```bash
-kubectl wait --for=condition=gardenletready --for=condition=extensionsready --for=condition=bootstrapped seed local --timeout=5m
-```
-
```

Comment:
Shouldn't we add an example for the `wait.sh` script to wait for the seed to be ready?
If not then adapt the next sentence (this doesn't read well anymore without this).

Actually, you have done this in the dev/ document.